ftplugin/scala/tagbar.vim

```diff
@@ -0,0 +1,49 @@
+"
+" Support for Tagbar -- https://github.com/majutsushi/tagbar
+"
+" Hat tip to Leonard Ehrenfried for the built-in ctags deffile:
+"    https://leonard.io/blog/2013/04/editing-scala-with-vim/
+"
+if !executable('ctags')
```

Comment:
The ctags executable may not be on the path (it's not on mine... don't ask why :p). I think this is probably overkill anyway.  If they've installed Tagbar, then it's probably been properly configured for ctags anyway.